checks/raw/shell_download_validate.go

```diff
@@ -696,6 +697,83 @@ func isChocoUnpinnedDownload(cmd []string) bool {
 	return true
 }
 
+func isUnpinnedNugetCliInstall(cmd []string) bool {
+	// Search for nuget commands.
+	if !isBinaryName("nuget", cmd[0]) {
+		return false
+	}
+
+	// Search for install commands.
+	if !strings.EqualFold(cmd[1], "install") {
+		return false
+	}
+
+	// package.config schema has required version field
+	// https://learn.microsoft.com/en-us/nuget/reference/packages-config#schema
+	if strings.HasSuffix(cmd[2], "packages.config") {
```

Comment:
so here the assumption is that users use a file named `<any>packages.config`, correct? Maybe add a comment about the assumption.

Reply:
added, thanks

Reply:
NuGet would only automatically discover `projectName.packages.config`